docs/browser.md

```diff
@@ -0,0 +1,54 @@
+---
+id: browser
+title: Browser
+---
+
+Run Prettier in the browser with the `standalone.js` UMD bundle shipped in the NPM package (starting in version 1.13). The new UMD bundle only formats the code and has no support for config and ignore files, no CLI usage and doesn't load plugins automatically.
+
+### `prettier.format(code, options)`
+
+Differently than the `format` function from the [main API](api.md#prettierformatsource-options), this function does not load plugins automatically, so a `plugins` property is required if you want to load external plugins. This also means the parsers included in the Prettier package won't be loaded automatically, so you need to load them before using.
```

Comment:
Doesn't load both internal and external plugins, so maybe drop the word "external"?